### PR TITLE
Spring Boot

### LINKED ISSUE #36: 🖼️ Framework Request: Spring Boot
| Question                                   | ✅ Yes | ❌ No |
|--------------------------------------------|-------|------|
| Is this framework already being requested? | []    | [x]   |

### What is the name of the framework?
Spring Boot
### What language does the framework use?
Java or Kotlin
### What is the website for the framework?
https://spring.io/projects/spring-boot

### PR DESCRIPTION
**Pull request by Evan-Nishi**
## Description
Added spring boot demo

## Related Issue
Closes #36 

## Motivation and Context
Adds spring boot in java

## Types of changes

- [x] New framework or language
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the structure and style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] There is no duplicate pull request for this fix/addition/issue resolution.
- [ ] If applicable, my code passes all current tests, and passes new tests that I have written to test it if the current tests do not suffice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,4 @@
 - ⚛️ [React](frameworks/react)
 - 🌶 [Flask](frameworks/flask)
 - 🦕 [Deno](frameworks/deno)
+- :seedling: [Spring Boot](frameworks/spring-boot)
```
Comment (Matt-Gleich) [inline README.md:17]: Please make this the actual emoji value

Comment (Cal-Hagner) [reply README.md:17]: Update: Please use !(Shields.io)[https://shields.io/] to make a badge to serve as the header for this wherever it needs to be.

Comment (cjdenio) [reply README.md:17]: Here's your badge URL.
https://img.shields.io/static/v1?label=Framework&message=Spring%20Boot&color=6DB33F&logo=spring&logoColor=white&style=for-the-badge

Comment (Cal-Hagner) [reply README.md:17]: Thank you @cjdenio.

**Comment by Evan-Nishi** [comment]
ok reformatted it and fixed typo, is it ok if I merge?

**Comment by cjdenio** [comment]
@Evan-Nishi I refactored the tree in your structure's README; please go through that and make sure everything's OK.

**Pull request merged**